README.md

```diff
@@ -56,35 +65,35 @@ $ vagrant ssh
 vagrant@vagrant:/vagrant$ ./scripts/server
 ```
 
-### Hot Reloading 🔥
-
-Because the frontend uses [Create React App](https://github.com/facebook/create-react-app/), which integrates with webpack, the page will automatically reload if you make changes to the code.
-
-In development, the [Django](https://www.djangoproject.com) app sits behind a [Gunicorn](https://www.gunicorn.org) worker that is passed the [`--reload` flag](https://docs.gunicorn.org/en/stable/settings.html#reload).
-
-### Development Data
+### Development Data 📈
 
-To destroy any existing development database and load fresh fixture data including users, facility lists, facility matches, and facilities run:
+To destroy the existing development database and load fresh fixture data including users, facility lists, facility matches, and facilities, run:
 
 ```bash
 vagrant@vagrant:/vagrant$ ./scripts/resetdb
 ```
 
+### Hot Reloading 🔥
+
+The frontend uses [Create React App](https://github.com/facebook/create-react-app/). When running `server`, the page will automatically [reload](https://github.com/facebook/create-react-app/#whats-included) if you make changes to the code.
+
+The [Django](https://www.djangoproject.com) app runs inside a [Gunicorn](https://www.gunicorn.org) worker. The worker will [restart](https://docs.gunicorn.org/en/stable/settings.html#reload) if you make changes to the code.
+
 ### Ports
 
 | Service                    | Port                            |
 | -------------------------- | ------------------------------- |
 | React development server   | [`6543`](http://localhost:6543) |
 | Gunicorn for Django app    | [`8081`](http://localhost:8081) |
 
-## Scripts
+## Scripts 🧰
```

Comment:
Let me know what you think about the use of emoji in the README. Does the color improve and make things seem more accessible, or is it distracting and silly?

Reply:
This is on brand for OAR. The newsletters are chock full of emoji. 👍